packages/api-bindings/src/graphql/common.graphql

```diff
@@ -116,6 +116,19 @@ fragment Wallet on Wallet {
   }
 }
 
+fragment Media on Media {
```

Comment:
I also dropped the `Fields` suffix while I was moving this definition from profile GQL to common GQL definitions file.

Reply:
I like it!